settings.gradle.kts

```diff
@@ -1,8 +1 @@
-rootProject.name = "LWCX"
-
-pluginManagement {
-    repositories {
-        mavenCentral()
-        gradlePluginPortal()
-    }
-}
\ No newline at end of file
+rootProject.name = "lwc"
```

Comment:
Why do you rename the project from LWCX to lwc? In combination with the changed group this will change a lot of things. E.g. the maven artifact will be `com.griefcraft:lwc:2.2.9-dev` instead of `com.griefcraft.lwc:LWCX:2.2.9-dev`.

And you have to manually rename the binary file.

Reply:
I think that's fine, we will just have to update the wiki.

The plugin has always been called LWC, LWCX (X for "Extended") is mainly used on the plugin page to distinguish us as a fork of the original with continued support. The original and other forks are no longer prevalent.